modules/dag-l1/src/main/scala/org/tessellation/dag/l1/cli/method.scala

```diff
@@ -52,20 +56,38 @@ object method {
     )
   }
 
+  val l0PeerIdOpts: Opts[PeerId] = Opts
+    .option[PeerId]("l0-peer-id", help = "L0 peer Id")
+    .orElse(Opts.env[PeerId]("CL_L0_PEER_ID", help = "L0 peer Id"))
+
+  val l0PeerHostOpts: Opts[Host] = Opts
+    .option[Host]("l0-peer-host", help = "L0 peer HTTP host")
+    .orElse(Opts.env[Host]("CL_L0_PEER_HTTP_HOST", help = "L0 peer HTTP host"))
+
+  val l0PeerPortOpts: Opts[Port] = Opts
+    .option[Port]("l0-peer-port", help = "L0 peer HTTP port")
+    .orElse(Opts.env[Port]("CL_L0_PEER_HTTP_PORT", help = "L0 peer HTTP port"))
+    .withDefault(port"9000")
+
+  val l0PeerOpts: Opts[L0Peer] =
```

Comment:
Should we put it in `L0Peer` object in shared? 🤔

Reply:
I don't think so. I'd put it into `sdk` rather than `shared`. Yet, I still stand for L0 peer hardcoding for genesis L1 node only and peer-discovery for the rest L1 nodes. Anyway, I can agree to have such a functionality as a middle-point to test L0/L1 interoperability.

Reply:
`opts` for classes like `Password` `KeyAlias`, live in companion objects in `shared` module, that's why I wonder. Would `org.tessellation.sdk.cli` and `peer` object make sense? 🤔